app/i18n/locales/nl.json

```diff
@@ -585,7 +585,7 @@
   "sending_email_confirmation": "e-mailbevestiging aan het verzenden",
   "Sending_to": "Verzenden naar",
   "Server": "Server",
-  "Server_version": "Server versie: {{version}}",
+  "Server_version": "Workspace: {{versie}}",
```

Comment:
I don't speak dutch, but `Server versie` looks more dutch than `workspace`.
Also you're translating the param to `versie`, so it's not going to work.